_rules/SC1-1-1-accessible-name-is-image-filename.md

```diff
@@ -0,0 +1,89 @@
+---
+name: Image accessible name is filename
+description: |
+  This rule checks that image elements that use their source filename as their accessible name do so without loss of infomation to the user.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to HTML `input` element with a [`type`](https://www.w3.org/TR/html/sec-forms.html#dom-htmlinputelement-type) of `image`, or `area` elements, or any HTML element with the semantic role of `img`, where the element is [included in the accessibility tree](#included-in-the-accessibility-tree) and has an [accessible name](#accessible-name) that is equivalent to the [filename](#filename) specified in the `src` attribute.
+
+**Note**: Difference in letter casing, and forward and trailing whitespace should be ignored. 
+
+### Expectation
+
+Each test target has an accessible name that serves an equivalent purposecan of the [non-text content](https://www.w3.org/TR/WCAG21/#dfn-non-text-content).
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility support
+
+ *There are no major accessibility support issues known for this rule.*
+
+## Background
+
+- [Understanding Success Criterion 1.1.1: Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
+- [F30: Failure of Success Criterion 1.1.1 and 1.2.1 due to using text alternatives that are not alternatives (e.g., filenames or placeholder text)](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/F30)
```

Comment:
the link to the F30 is a not stable one (TR/2016/NOTE-WCAG20-TECHS-20161007). Do we better link to techniques WCAG 2.1 document : https://www.w3.org/WAI/WCAG21/Techniques/failures/F30 ?

Reply:
Nice one! great spot :)